example.fi

```diff
@@ -69,3 +69,8 @@ end
 fn not(a: Bool) : Bool do
   !a
 end
+
+# Unary minus operator
+fn negate(a: Int) : Int do
+  -a
+end
```

Comment:
Let's use "unary_minus" as the function name since negate also means logical "not"

Reply:
Also, let's add a newline at the end of file to be consistent